pkg/metrics/metrics.go

```diff
@@ -29,21 +29,23 @@ var (
 			Namespace: Namespace,
 			Subsystem: nodeSubsystem,
 			Name:      "created",
-			Help:      "Number of nodes created in total by Karpenter. Labeled by reason the node was created.",
+			Help:      "Number of nodes created by the given provisoner in total by Karpenter. Labeled by reason the node was created and the owning provisioner.",
```

Comment:
```suggestion
			Help:      "Number of nodes created in total by Karpenter. Labeled by reason the node was created and the owning provisioner.",
```

nit: feels redundant to say provisioner twice here